addons/sourcemod/scripting/vsh/bosses/boss_yeti.sp

```diff
@@ -0,0 +1,164 @@
+	"ambient_mp3/lair/animal_call_yeti3.mp3", 
+	"ambient_mp3/lair/animal_call_yeti4.mp3", 
+	"ambient_mp3/lair/animal_call_yeti5.mp3", 
+	"ambient_mp3/lair/animal_call_yeti6.mp3", 
+	"ambient_mp3/lair/animal_call_yeti7.mp3"
+};
+
+static char g_strYetiWin[][] =  {
+	"player/taunt_yeti_roar_first.wav"
+};
+
+static char g_strYetiLose[][] =  {
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiKill[][] =  {
+	"player/taunt_yeti_roar_beginning.wav"
+};
+
+static char g_strYetiLastMan[][] =  {
+	"ambient_mp3/lair/animal_call_yeti2.mp3", 
+};
+
+static char g_strYetiBackStabbed[][] =  {
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiVoice[][] =  {
+	"player/taunt_yeti_roar_beginning.wav",
+	"player/taunt_yeti_roar_first.wav", 
+	"player/taunt_yeti_roar_second.wav"
+};
+
+static char g_strYetiFootsteps[][] =  {
+	"player/footsteps/giant1.wav", 
+	"player/footsteps/giant2.wav"
+};
+
+methodmap CYeti < SaxtonHaleBase
+{
+	public CYeti(CYeti boss)
+	{
+		boss.CallFunction("CreateAbility", "CBraveJump");
+		boss.CallFunction("CreateAbility", "CGroundPound");
+		boss.CallFunction("CreateAbility", "CRageFreeze");
+		CModelOverride modelOverride = boss.CallFunction("CreateAbility", "CModelOverride");
+		modelOverride.SetModel(YETI_MODEL);
+		
+		boss.iBaseHealth = 800;
+		boss.iHealthPerPlayer = 850;
+		boss.nClass = TFClass_Heavy;
+		boss.iMaxRageDamage = 2500;
+	}
+	
+	public void GetBossName(char[] sName, int length)
+	{
+		strcopy(sName, length, "Last Yeti");
+	}
+	
+	public void GetBossInfo(char[] sInfo, int length)
+	{
+		StrCat(sInfo, length, "\nHealth: High");
+		StrCat(sInfo, length, "\n ");
+		StrCat(sInfo, length, "\nAbilities");
+		StrCat(sInfo, length, "\n- Brave Jump");
+		StrCat(sInfo, length, "\n- Ground Pound (Passive)");
+		StrCat(sInfo, length, "\n ");
+		StrCat(sInfo, length, "\nRage");
+		StrCat(sInfo, length, "\n- Slows players at medium range for 3 seconds");
+		StrCat(sInfo, length, "\n- Affected players get frozen for 4 seconds");
+		StrCat(sInfo, length, "\n- 200%% Rage: Extended range and freeze duration increased to 6 seconds");
+	}
+	
+	public void OnSpawn()
+	{
+		char attribs[128];
+		Format(attribs, sizeof(attribs), "2 ; 2.80 ; 252 ; 0.5 ; 259 ; 1.0 ; 214 ; %d", GetRandomInt(7500, 7615));
+		int iWeapon = this.CallFunction("CreateWeapon", 195, NULL_STRING, 100, TFQual_Strange, attribs);
+		if (iWeapon > MaxClients)
+			SetEntPropEnt(this.iClient, Prop_Send, "m_hActiveWeapon", iWeapon);
+		/*
+		Fist attributes:
+		
+		2: damage bonus
+		252: reduction in push force taken from damage
+		259: Deals 3x falling damage to the player you land on
+		214: kill_eater
+		*/
+	}
+	
+	public void OnThink()
+	{
+		int iWeapon = GetPlayerWeaponSlot(this.iClient, view_as<int>(WeaponSlot_Melee));
+		if (iWeapon > MaxClients)
+			SetEntPropFloat(iWeapon, Prop_Send, "m_flNextSecondaryAttack", GetGameTime() + 60.0);
+	}
+	
+	public void OnRage()
+	{
+		FakeClientCommand(this.iClient, "voicemenu %d %d", 2, 1);
```

Comment:
Any reason to format number with `%d`? could've just directly went for `voicemenu 2 1`

Reply:
No real reason. I just figured if the command has arguments I might as well specify them like this